packages/client/ui/react-ui/src/lib/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { type ClassValue, clsx } from "clsx";
+import { twMerge } from "tailwind-merge";
+
+/**
+ * Combines multiple class names into a single string.
+ *
+ * @param inputs - The class names to combine.
+ * @returns The combined class names as a string.
+ */
+export function cn(...inputs: ClassValue[]) {
```

Comment:
use full name `classNames` to match crossmint-main

Reply:
fixed 👍